modules/seqexec/server/src/main/scala/seqexec/server/tcs/Tcs.scala

```diff
@@ -104,6 +104,28 @@ object Tcs {
   val P_OFFSET_PROP: String = "p"
   val Q_OFFSET_PROP: String = "q"
 
+  sealed trait CRFollow
+  object CRFollow {
+    case object On extends CRFollow
+    case object Off extends CRFollow
+
+    def keywordValue(cr: CRFollow): String = cr match {
+      case On  => "yes"
+      case Off => "no"
+    }
+
+    def fromInt: Prism[Int, CRFollow] =
+      Prism[Int, CRFollow] {
+        case 0  => Off.some
+        case 1  => On.some
+        case _  => none
+      } {
+        case Off => 0
+        case On  => 1
+      }
+
+  }
+
```

Comment:
I am not sure this class is the right place for this piece of code.

Reply:
Do you mean the whole CRFollow or just the prism. The Prism is used to decode the epics values